client/src/pages/about.js

```diff
@@ -31,8 +16,8 @@ const AboutPage = () => {
         <Row>
           <Col
             className='questions'
-            md={6}
-            mdOffset={3}
+            md={8}
```

Comment:
Hi @ahmadabdolsaheb this change should be reverted. I think it was intentional to have the narrower width as we have on production.

Reply:
```suggestion
            md={6}
```

Please revert this back to what it was.